src/components/admin/MainIssueController.jsx

```diff
@@ -240,7 +445,7 @@ export default class MainIssueController extends FalcorController {
             secondary
             style={styles.publishingButtons}
             onTouchTap={this.unpublishIssue}
-            disabled={!published}
+            disabled={!published || this.state.changed || this.state.saving}
```

Comment:
Let's also do it if we're publishing, that should already have been here, not your bad but lets fix it now